src/index.ts

```diff
@@ -143,9 +143,42 @@ export class WalletPluginAnchor implements WalletPlugin {
      */
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     async sign(resolved: ResolvedSigningRequest, context: TransactContext): Promise<Signature> {
+        // Create the identity request to be presented to the user
+        const {callback, request} = await createTransactionRequest(context)
+
+        // Tell Wharf we need to prompt the user with a QR code and a button
+        context.ui.prompt({
+            title: 'Sign transaction in Anchor',
+            body: 'Scan the QR-code with Anchor on another device or use the button to open it here.',
+            elements: [
+                {
+                    type: 'qr',
+                    data: String(request),
+                },
+                {
+                    type: 'button',
+                    label: 'Open Anchor',
+                    data: String(request),
+                },
+            ],
+        })
+
+        if (session) {
+            session.transact(request)
+        }
+
+        redirectToAnchor()
```

Comment:
This process currently assumes you're logged in with an account already and have a session. This function call is happening from within a session (like I described above), so what it needs to do here is take the request and sent it using the buoy client to the wallet that's already listening.

Reply:
A fallback UI here though could be useful, if the wallet isn't responding.

Reply:
You do want to send the request via buoy, but don't you also want to take the user to the Anchor app? Or is the session kit going to handle that?